tests/test_age_pyramid.py

```diff
@@ -8,31 +8,36 @@
 
 from eds_scikit.datasets.synthetic.person import load_person
 from eds_scikit.plot.data_quality import plot_age_pyramid
+import pandas as pd 
+import numpy as np
 
 data = load_person()
 
+person_with_inclusion_date = data.person.copy()
+person_with_inclusion_date["inclusion_datetime"] = person_with_inclusion_date["birth_datetime"] + pd.to_timedelta(np.random.randint(0, 1000, len(person_with_inclusion_date)), unit='d')
 
-def test_plot_age_pyramid():
-    original_person = data.person.copy()
-
-    datetime_ref = datetime(2020, 1, 1)
-    chart, group_gender_age = plot_age_pyramid(data.person, datetime_ref, savefig=False)
+@pytest.mark.parametrize(
```

Comment:
Could you split this test into two functions?
1. The first one (this function) asserts that `datetime_ref` is correct. You should also add a parseable date string in the mark input.
2. The second one checks that errors are returned from incorrect `datetime_ref` (all error cases should be tested to maximize test coverage) by leveraging `pytest.raises`.

Reply:
new test added ! Thanks for the review :heart: